finalPrep/modZeroHero.js

```diff
@@ -55,30 +80,78 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
-
+var scaryMonster = {
+  name: "The Ooze",
+  smell: "bad",
+  weight: 2000,
+  citiesDestroyed: ["Jamestown", "Kingston"],
+  luckyNumbers: [1, 3, 7],
+  address: {
+    number: 123,
+    street: "Main",
+    state: "New York",
+    zip: 10016
+  }
+};
+console.log(scaryMonster);
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
 
 // - Create the following class methods
 //   - sayName, should print the hero's name to the console
 //   - maximizeEnergy, should update the energyLevel to 1000
 //   - gainPower, should take an argument of a number and INCREASE the powerLevel by that number
+class SuperHero {
```

Comment:
Great work with this - everything is spot on. 

Another little clean up - deleting line 121.